topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: tutorial_hands_on
+
+title: "NCBI BLAST+ against the MAdLand"
+zenodo_link: 'https://zenodo.org/record/4710649'
+questions:
+- "How to access Galaxy server?"
+- "How can we perform Blast analysis on Galaxy?"
+- "What is MAdLand DB?"
+
+objectives:
+- "Load fasta sequence into Galaxy"
+- "Perform NCBI-Blast+ analysis on Galaxy"
+
+time_estimation: "20m"
+key_points:
+- Blast tool searches a database of sequences for similar sequences to a query sequence.
+- MAdLand is a database of fully sequenced plant and algal genomes, with an emphasis on non-seed plants and streptophyte algae that can be use for sequence similarity search.
+
+contributors:
+- deeptivarshney
+
+---
+
+# Introduction
+
+<!-- This is a comment. -->
+
```

Comment:
As I mentioned to you, it would be nice to design the tutorial with the objective of solving a specific question/biological problem. In which context do you think will be useful to use that database? Which scientific question could be addressed by using it?